### PR TITLE
add_embedding: added support for multiple labels 

### PR DESCRIPTION
**Pull request by h0rm**
using list of list for metatdata and a metadata_header parameter multiple labels can be stored for the embedding.

### CODE REVIEW

**Comment by lanpa** [comment]
Thanks. Can you provide a simple example?

**Comment by h0rm** [comment]
Taking your MNIST example you could for instead be interested in comparing the training set with the testset. This is can be done by adding the following lines to your example:

```python
dataset = datasets.MNIST('mnist', train=True, download=True)
images_train = dataset.train_data[:100].float()
labels_train = dataset.train_labels[:100]
features_train = images_train.view(100, 784)

all_features = torch.cat((features, features_train))
all_labels = torch.cat((label, labels_train))
all_images = torch.cat((images, images_train))
dataset_label = ['test']*100 + ['train']*100
all_labels = list(zip(all_labels, dataset_label))

writer.add_embedding(all_features, metadata=all_labels, label_img=all_images.unsqueeze(1),
                     metadata_header=['digit', 'database'])
```

This adds two labels in tensorboard 'digit' and 'database'. We use this for example to additionally visualize the writers (which are not available for MNIST). In general you can use this for any multi-label dataset.

**Comment by lanpa** [comment]
thanks @h0rm . I didn't know this hidden feature in tensorboard before lol.
